optuna/storages/_rdb/storage.py

```diff
@@ -64,6 +65,46 @@
 _logger = optuna.logging.get_logger(__name__)
 
 
+def _float_without_nan_to_pair_repr(value: float) -> Tuple[float, models.FloatTypeEnum]:
```

Comment:
These methods are only used in RDBStorage. How about making them as `staticmethod`s of RDBStorage?

Reply:
:+1: I'll fix it. Thanks.